zebra-rpc/src/methods/get_block_template_rpcs/types/get_mining_info.rs

```diff
@@ -0,0 +1,26 @@
+//! Response type for the `getmininginfo` RPC.
+
+use zebra_chain::parameters::Network;
+
+/// Response to a `getmininginfo` RPC request.
+#[derive(Debug, PartialEq, Eq, serde::Serialize)]
+pub struct Response {
+    /// The estimated network solution rate in Sol/s.
+    networksolps: u128,
+
+    /// The estimated network solution rate in Sol/s.
+    networkhashps: u128,
+
+    /// Current network name as defined in BIP70 (main, test, regtest)/
+    chain: Network,
```

Comment:
It would be really easy to add the `testnet` field here as well, what do you think?

Reply:
I think so too. Added in [`190b622`](https://github.com/ZcashFoundation/zebra/pull/5808/commits/190b622ae96a905f6e5235bc7d63a4dbb879b290).